sys/linux/sys.txt

```diff
@@ -264,6 +264,8 @@ mq_unlink(name ptr[in, string])
 
 mknod(file ptr[in, filename], mode flags[mknod_mode], dev int32)
 mknod$loop(file ptr[in, filename], mode flags[mknod_mode], dev proc[1792, 2])
+mknodat$loop(dirfd fd_dir, file ptr[in, filename], mode flags[mknod_mode], dev proc[1792, 2])
+mknodat$null(dirfd fd_dir, file ptr[in, filename], mode flags[mknod_mode], dev proc[256, 3])
```

Comment:
`proc` type won't yield 0x103, it allocates per-test private regions of values. I think we need `const[0x103]` here.
Why do we need /dev/null here? Could we open `/dev/null` instead (`openat$null`)? Or we need this file in the test dir?

Reply:
> `proc` type won't yield 0x103, it allocates per-test private regions of values. I think we need `const[0x103]` here.

Right, `const[0x103]` is better. Where can we find the documentation for `proc`? I misinterpreted its meaning. How does `proc[1792, 2]` translates?

> Why do we need /dev/null here? Could we open `/dev/null` instead (`openat$null`)? Or we need this file in the test dir?

Being able to create `/dev/null` enables to go through the Landlock character device creation code (`LANDLOCK_ACCESS_FS_MAKE_CHAR`).

Reply:
https://github.com/google/syzkaller/blob/master/docs/syscall_descriptions_syntax.md#proc

Reply:
OK, it makes sense now. :)
Should I remove the variants or use fixed values (instead of `proc`) for `mknodat$loop` and `mknod$null`?

Reply:
For /dev/null we need to use `const[0x103]`. For loop we want to keep existing proc because we want to give each test process own loop devices so that they don't interfere with each other.